### PR TITLE
Container version update

### PR DESCRIPTION
**Pull request by mjurc**
* Removing microrelease version numbers from image tags to pick up
  latest minors
* Bumping RHSSO 7.4 to 7.5
* Bumping RHSCL PostgreSQL to 13
* Bumping Oracle XE to 21

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -199,20 +199,20 @@
                                 <ts.global.s2i.quarkus.native.builder.image>${ts.global.s2i.quarkus.native.builder.image}</ts.global.s2i.quarkus.native.builder.image>
                                 <!-- Services used in test suite -->
                                 <keycloak.image>quay.io/keycloak/keycloak:15.0.1</keycloak.image>
-                                <rhsso.73.image>registry.access.redhat.com/redhat-sso-7/sso73-openshift</rhsso.73.image>
-                                <postgresql.13.image>quay.io/bitnami/postgresql:13.4.0</postgresql.13.image>
-                                <mysql.57.image>quay.io/bitnami/mysql:5.7.32</mysql.57.image>
+                                <rhsso.image>registry.access.redhat.com/rh-sso-7/sso75-openshift-rhel8</rhsso.image>
+                                <postgresql.13.image>quay.io/bitnami/postgresql:13.5</postgresql.13.image>
+                                <mysql.57.image>quay.io/bitnami/mysql:5.7</mysql.57.image>
                                 <mysql.80.image>registry.access.redhat.com/rhscl/mysql-80-rhel7</mysql.80.image>
```
Comment (rsvoboda) [inline pom.xml:205]: Use `rhscl` as in case of mariadb

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rsvoboda) [reply pom.xml:205]: And let's see how CI ends, maybe we will need secrets to pull from RH registry

**Comment by rsvoboda** [comment]
@mjurc CI failures, please investigate if they are related to the PR

File: pom.xml (hunk 1 above)
Comment (mjurc) [reply pom.xml:205]: I'm gonna resolve this as it is irrelevant for now

**Comment by mjurc** [comment]
run tests

**Comment by mjurc** [comment]
I've rerun the affected tests on OCP manually and the failures were unrelated.

Ping me for CI runs

**Comment by rsvoboda** [comment]
@mjurc CI failed

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**